diesel/src/mysql/backend.rs

```diff
@@ -4,16 +4,17 @@ use backend::*;
 use query_builder::bind_collector::RawBytesBindCollector;
 use super::query_builder::MysqlQueryBuilder;
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
 pub struct Mysql;
 
-#[allow(missing_debug_implementations, missing_copy_implementations)]
+#[allow(missing_debug_implementations)]
```

Comment:
Honestly, no real reason. `#[derive()]` feels more attached to the item, while `#[allow]` feels more attached to the block of code.